src/main/app/common/allowances/allowanceRepository.ts

```diff
@@ -0,0 +1,73 @@
+import { AllowanceItem } from 'common/allowances/allowanceItem'
+import { Allowance } from 'common/allowances/allowance'
+import { join } from 'path'
+
+export enum DependantAllowanceType {
```

Comment:
I would have places those types in the `allowance` module instead - what do you think?